components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -31,11 +31,19 @@ function ProfileDetailCard({
         <p className=" font-bold "> {name}</p>
       </div>
       <div className="flex flex-col ml-4 mb-4 gap-4 items-start">
-        <JoinedDateComponent user={name} createdAt={joinDate} />
+        <JoinedDateComponent
+          user={name}
+          textSize={"small"}
```

Comment:
these changes are from the redux branch that this branch was based off of. I was missing the textSize={"small"} prop which I added and reformatted.

Reply:
Great job, leaving that level of context!